consensus/src/payload_manager/quorum_store_payload_manager.rs

```diff
@@ -119,6 +129,47 @@ impl QuorumStorePayloadManager {
         }
         Ok(all_txns)
     }
+
+    fn batches_in_block(block: &Block) -> Vec<BatchInfo> {
+        if let Some(payload) = block.payload() {
+            payload.get_all_batches()
+        } else {
+            vec![]
+        }
+    }
+
+    fn get_committed_timestamp_and_round(
```

Comment:
Semantics nit: This function name is confusing to me since i think of `commit_root` for `get_committed_timestamp_and_round`, but in reality we're using the `window_root`